regex-capi/src/macros.rs

```diff
@@ -18,10 +18,7 @@ macro_rules! ffi_fn {
                     } else {
                         "UNABLE TO SHOW RESULT OF PANIC.".to_owned()
                     };
-                    let _ = writeln!(
-                        &mut io::stderr(),
-                        "panic unwind caught, aborting: {:?}",
-                        msg);
+                    let _ = writeln!(&mut io::stderr(), "panic unwind caught, aborting: {msg:?}");
```

Comment:
I follow a 79 column (inclusive) rule. I'm kind of surprised `rustfmt` doesn't take care of this here... Oh, because it's in a macro. So it will need to be wrapped manually.